src/eko/couplings.py

```diff
@@ -477,7 +477,7 @@ def compute(self, a_ref, nf, scale_from, scale_to):
         """
         key = (float(a_ref[0]), float(a_ref[1]), nf, scale_from, float(scale_to))
         try:
-            return self.cache[key]
+            return self.cache[key].copy()
```

Comment:
At the moment the cache is just empty, so I'm pretty sure you want to fill it.

In case the content will be a structured object, pay attention to the difference between a deep and shallow copy. In any case, sometimes even a shallow copy is just fine, but you have to be aware and do it on purpose. If it's not structured, never mind.